rosidl_typesupport_freertps_cpp/cmake/rosidl_typesupport_freertps_cpp_generate_interfaces.cmake

```diff
@@ -281,6 +170,8 @@ add_dependencies(
 )
 
 if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
+  ament_export_libraries(${rosidl_generate_interfaces_TARGET}${_target_suffix} ${freertps_LIBRARIES})
```

Comment:
Does the generated code depend on `freertps_LIBRARIES`? If yes, shouldn't the library `${rosidl_generate_interfaces_TARGET}${_target_suffix}` be linked against it? If no, why should this package export it as a dependency?